src/Steam.Common/App.cs

```diff
@@ -8,42 +8,53 @@ namespace Steam
     public readonly struct App : IEquatable<App>
     {
         /// <summary>
-        /// The Id
+        /// The value of the app id
         /// </summary>
-        public readonly uint Id;
+        public readonly uint Value;
 
         /// <summary>
         /// Invalid app value
         /// </summary>
-        public static readonly App Invalid = 0x0;
+        public static readonly App Invalid = (App)0x0;
+
+        /// <summary>
+        /// The max value of a uint24 data structure
+        /// </summary>
+        public const uint UInt24MaxValue = 16777215;
```

Comment:
Possibly just call it MaxValue since it *could* change in the future (app IDs are being generated today in the 10,000,000s range).